.clang-format

```diff
@@ -38,56 +60,83 @@ BraceWrapping:
   AfterExternBlock: false
   BeforeCatch:     true
   BeforeElse:      true
+  BeforeLambdaBody: false
+  BeforeWhile:     false
   IndentBraces:    false
   SplitEmptyFunction: false
   SplitEmptyRecord: false
   SplitEmptyNamespace: false
 BreakBeforeBinaryOperators: None
+BreakBeforeConceptDeclarations: Allowed
 BreakBeforeBraces: Custom
 BreakInheritanceList: BeforeColon
 BreakBeforeTernaryOperators: true
 BreakConstructorInitializers: BeforeColon
 BreakStringLiterals: false
 ColumnLimit:     105
 CommentPragmas:  '^ IWYU pragma:'
+QualifierAlignment: Left
 CompactNamespaces: false
-ConstructorInitializerAllOnOneLineOrOnePerLine: false
 ConstructorInitializerIndentWidth: 4
 ContinuationIndentWidth: 4
 Cpp11BracedListStyle: true
 DeriveLineEnding: false
 DerivePointerAlignment: false
 DisableFormat:   false
+EmptyLineAfterAccessModifier: Never
+EmptyLineBeforeAccessModifier: Always
+PackConstructorInitializers: BinPack
 FixNamespaceComments: false
 ForEachMacros:
   - foreach
   - Q_FOREACH
   - BOOST_FOREACH
-IncludeBlocks:   Preserve
+IfMacros:
+  - KJ_IF_MAYBE
+IncludeBlocks:   Regroup
 IncludeCategories:
-  - Regex:           'dlaf.*/'
+  - Regex:           '^<dlaf_test'
```

Comment:
May I ask you to sort the categories by priority (in the same order they would appear in source files, so priority 1 first?)?

Reply:
Not possible, the order is the evaluation order and is some cases changing the order produce different groups

Reply:
E.g. 
```
  - Regex:           '^<.*>'
    Priority:        2
```
will match almost all include headers.

Reply:
Makes sense, thanks!